sys/kern/src/arch/rv64/stimer.rs

```diff
@@ -4,7 +4,7 @@
 
 use crate::arch::sbi_set_timer;
 
-const HAS_SSTC: bool = true; // TODO: Runtime detection
+const HAS_SSTC: bool = false; // TODO: Runtime detection
```

Comment:
Is there a JIRA issue already raised to capture this TODO?  We can put in the link to that JIRA issue here for tracking.

Reply:
Do we want runtime detection rather than just a kernel config?  We have chosen kernel configs for other hardware features like vectored interrupts.

Reply:
My suspicion is that we might have to use kernel config here as well since there might not be an easy way to probe for this feature unless we have some mechanism to expose this information from M-mode to S-mode.

Reply:
I think the app developer shouldn't really be responsible for figuring this out. It should be an attribute of the chip itself. Anyways, there is more work required for getting to that point, but yeah for now a static config feature selected by app seems fine.

Reply:
Actually yeah, that is more sensible. The way the `chip.toml` is currently used doesn't really lend itself to this sort of configuration, but it's certainly possible if it's something we want for the future.

Reply:
As we are pushing more and more stuff into various config tomls and cargo features, would it make sense to look into using something like device trees to bring everything all the hardware config to one place?